### ISSUE TITLE
Support placement zones & isolation segments

### ISSUE THREAD

**Issue by btat** [SUSE/doc-cap#349]
https://trello.com/c/zZmx9YQg/915-5-support-placement-zones-isolation-segments

**Comment by cjschroder** [comment]
I really want to close this because it is not ready for docs. We need a reliable tested source document to work from. @btat what do you think? We have too many open Issues that are not doc-ready.

**Comment by btat** [comment]
My copy pasta from #357 

```
This (and the other similar not quite ready for docs issues opened) is probably me jumping the gun a bit on creating the issues. I thought it'd be nice to have on our radar/tracking board of upcoming items tagged as needing docs attention from the sprint board (and in case we forget to create an issue later down the line when the items are ready). I agree though, we should discuss this during the meeting. Will you be joining this week's meeting since you're at the offsite our save this for the following week?
```

Also marking as blocked until the card itself is unblocked.
